ci/properties/matlab.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "MATLAB",
+    "description": "Build and test with MATLAB",
+    "iconName": "matlab",
+    "creator": "MathWorks",
+    "categories": [
+        "MATLAB",
+        "Continuous integration",
+        "testing"
```

Comment:
I don't see any other properties files with this category. Where did it come from?

Reply:
It's listed [here](https://github.com/actions/starter-workflows#categories)

Reply:
Interesting. That list almost seems outdated because I don't see most of those categories in use by other templates and the "continuous-integration" category is "Continuous integration" (without the dash) in all of the templates I see. I guess we'll find out when we do a pull to actions/starter-workflows.